### ISSUE TITLE
Improvements to accessibility label of not available credentials in proof request

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1432]
### Description of feature / user story
Currently, attributes/credentials that are not available to satisfy a proof request are read by a screen reader as "undefined" twice and the "not available" label is not read. This isn't clear to person with a visual disability that they have a credential missing.

### Acceptance Criteria
- "Not available" should be read by screen reader i.e. "[Credential name] not available"
- "undefined" accessibility label should be changed to "not available" i.e. "[attribute label] not available"

### Wireframes or relevant image assets / links
